packages/vertexai/src/methods/chat-session-helpers.ts

```diff
@@ -48,28 +54,32 @@ export function validateChatHistory(history: Content[]): void {
   for (const currContent of history) {
     const { role, parts } = currContent;
     if (!prevContent && role !== 'user') {
-      throw ERROR_FACTORY.create(VertexError.INVALID_CONTENT, {
-        message: `First content should be with role 'user', got ${role}`
-      });
+      throw new VertexAIError(
+        VertexAIErrorCode.INVALID_CONTENT,
+        `First content should be with role 'user', got ${role}`
```

Comment:
We should probably capitalize Content here also even though we missed it earlier.